src/server/system_services/bucket_server.js

```diff
@@ -1142,20 +1142,32 @@ function get_cloud_buckets(req) {
                 return storage.getBuckets()
                     .then(data => data[0].map(bucket =>
                         _inject_usage_to_cloud_bucket(bucket.name, connection.endpoint, used_cloud_buckets)));
-            } else { //else if AWS
+            } else {
```

Comment:
can you return the comment as well - seems valuable especially after we forgot compatible.
maybe add this // else if AWS(s3-compatible/aws/sts-aws)

Reply:
maybe all the list in 1168: AWS', 'S3_COMPATIBLE', 'FLASHBLADE', 'IBM_COS'

Reply:
Done